tools/export-translations.php

```diff
@@ -154,12 +154,13 @@ function apize_url( $url ) {
 	}
 
 	$output_file = "{$temp_file_path}/jetpack-{$map[$full_locale]}.po";
-	$input_url   = sprintf( '%s/%s/%s/export-translations?format=po', $source_url, $set->locale, $set->slug );
-	$exec        = sprintf( 'curl -s -o %s %s', escapeshellarg( $output_file ), escapeshellarg( $input_url ) );
+	$input_url   = sprintf( '%s/%s/%s/export-translations/?format=po', $source_url, $set->locale, $set->slug );
+	$exec        = sprintf( 'curl -sL -o %s %s', escapeshellarg( $output_file ), escapeshellarg( $input_url ) );
```

Comment:
Would the following be more clear? No need to be succinct in a script.

```suggestion
	$exec        = sprintf( 'curl --silent --location --output %s %s', escapeshellarg( $output_file ), escapeshellarg( $input_url ) );
```

Reply:
I like that.